sample.py

```
@@ -0,0 +1,26 @@
+
+A=1
+A=1
+def DoSomething(x,y):return x+y+ A 
+
+class myclass:
+	def __init__(self):self.Data=[1,"two",3.0];self.v=42 
+
+	def run(self):print(self.Data);print(DoSomething(5,7))
+
+def main(): 
+	for i in range(5):print(i*2+3)
```

Ruff (0.14.8)

Multiple statements on one line (colon)

(E701)